$OpenBSD: patch-share_pnp_templates_dist_check_multi_php,v 1.1 2016/05/17 07:13:42 kirby Exp $

Fix compatibility with rrdtool-1.6.x
Upstream commit 0a539e1c7ab03ec015c27e4ad0fe16343a98c269

--- share/pnp/templates.dist/check_multi.php.orig	Mon May 16 16:41:57 2016
+++ share/pnp/templates.dist/check_multi.php	Mon May 16 16:42:13 2016
@@ -4,7 +4,7 @@
 # Template for check_multi
 # 
 
-$opt[1] = "--lower=$MIN[1] --vertical-label num  --title \"Number of Checks\" ";
+$opt[1] = "--lower-limit=$MIN[1] --vertical-label num  --title \"Number of Checks\" ";
 $ds_name[1] = "Executed Plugins";
 
 $def[1] =  "DEF:var1=$RRDFILE[1]:$DS[1]:AVERAGE " ;
